docs/tutorials/custom-commit-notifications.md

```diff
@@ -0,0 +1,124 @@
+Things happen all the time in your development workflow. Commits are pushed,
+builds are started, they fail or succeed. Team members, or even users, open and
+comment issues everyday.
+
+All that flow of information can become overwhelming. With Atomist, you can
+attach behavior to these events and automate all sorts of near real-time
+responses.
+
+These tutorials show you how to rely on Atomist to automate those scenarios.
+
+!!! tip "All you need is Rug"
+    These tutorials aim at showing you the code that you could write to
+    automate event reactions. They assume you have a [Rug project][ugpj]. You
+    can easily add [Rug event handler][rugev] to that project via the public 
+    [AddTypeScriptEventHandler Rug editor][rugeditor] provided by Atomist.
+
+[rugev]: /user-guide/rug/handlers.md
+[ugpj]: /user-guide/rug/projects.md
+[rugeditor]: https://github.com/atomist/rug-editors#addtypescripteventhandler
+
+## Getting Notified from a User Commit
+
+The first events you will ask Atomist to react to are commits pushed by a user.
+To achieve this, you add a [Rug event handler][rugev] to your
+[Rug project][ugpj] that, once you have [published][ugpub] it to your team, will
+be evaluated and executed by Atomist on such events.
+
+[ugpub]: /user-guide/rug/lifecycle.md#publishing
+
+Below is such an event handler:
+
+```typescript linenums="1"
+{!../../.atomist/handlers/event/NewCommitPushedToGeneral.ts!}
+```
+
+An event handler tells Atomist what it is interested in via its
+[path expression][ugpxe] on line 10. Notice in particular how the handler
```

Comment:
Lines 11-13 when I build the site. I would reword this a bit to "via its path expression, which we see on lines 11--13".